site/docs/v1/tech/shared/_scim-limits.adoc

```diff
@@ -0,0 +1,7 @@
+The current SCIM implementation does not support:
+
+* List operations
```

Comment:
I'd add bulk operations to this list too. 
https://datatracker.ietf.org/doc/html/rfc7644#section-3.7

Reply:
Thanks, added!